Makefile

```diff
@@ -48,3 +48,15 @@ run:
 
 check:
 	pytest-3 --color=yes --showlocals -vv $(TEST_TARGET)
+
+check-in-container:
```

Comment:
Should this be phony and depend on `build`?

Reply:
Since we are binding the current code to the container I would go with the manual `build` to save some time on each run.